.travis.yml

```diff
@@ -14,6 +16,10 @@ matrix:
     - php: 7.0
   fast_finish: true
 
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
You MUST run with `--prefer-dist` for install now, or this is totally pointless.